kanidmd/src/lib/value.rs

```diff
@@ -237,6 +243,7 @@ impl fmt::Display for SyntaxType {
                 SyntaxType::Cid => "CID",
                 SyntaxType::NsUniqueId => "NSUNIQUEID",
                 SyntaxType::DateTime => "DATETIME",
+                SyntaxType::EmailAddress => "EMAIL_ADDRESS",
```

Comment:
Since it's only writing a `&str` we should just use `f.write_str(match self { ... })` to clear up lines 226-227.

Reply:
That's an awesome suggestion :)